packages/valory/skills/price_estimation_abci/fsm_specification.yaml

```diff
@@ -19,6 +19,7 @@ transition_func:
     (CollectObservationRound, NO_MAJORITY): CollectObservationRound
     (CollectObservationRound, ROUND_TIMEOUT): CollectObservationRound
     (EstimateConsensusRound, DONE): TxHashRound
+    (EstimateConsensusRound, NONE): CollectObservationRound
```

Comment:
what is the difference between returning None and `Event.NONE`? Should we make it such that we always return `Event.NONE` in case we don't branch into another execution path first?

Reply:
Returning `None` means that the execution of the round is still going on. We simply re-enter. Returning something other than `None` means that the round ended.